tests/unit/components/ParameterForm.spec.ts

```diff
@@ -1,9 +1,12 @@
 import ParameterForm from "@/components/ParameterForm.vue";
 import { emptyScenario, mockedMetadata, mockPinia, updatableNumericParameter } from "@/tests/unit/mocks/mockPinia";
-
+import { CButtonGroup, CTooltip } from "@coreui/vue";
 import { mockNuxtImport, mountSuspended, registerEndpoint } from "@nuxt/test-utils/runtime";
 import { flushPromises } from "@vue/test-utils";
 import { FetchError } from "ofetch";
+import { beforeEach, describe, expect, it, vi } from "vitest";
```

Comment:
We don't need to import anything from vitest ever - it's automatically imported.

Reply:
```suggestion
```